From 2060ebc35b03ffdc976331c6f87ff21bee5acb60 Mon Sep 17 00:00:00 2001
From: Jun Miao <jun.miao@windriver.com>
Date: Wed, 29 Jul 2020 20:22:40 +0800
Subject: [PATCH] KVM: cpuid: Fix a build warning

This build warning is casued by 77da19613af0 (KVM: cpuid: extract
do_cpuid_7_mask and support multiple subleafs), Which is a backporting
(commit 50a9e1a4b1dece60fd79ecdee25db01274a7f291 upstream), but lost
four lines. So delete them.

Signed-off-by: Jun Miao <jun.miao@windriver.com>
---
 arch/x86/kvm/cpuid.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/x86/kvm/cpuid.c b/arch/x86/kvm/cpuid.c
index 35f8c232827a..d85a9d6985c4 100644
--- a/arch/x86/kvm/cpuid.c
+++ b/arch/x86/kvm/cpuid.c
@@ -473,10 +473,6 @@ static inline int __do_cpuid_func(struct kvm_cpuid_entry2 *entry, u32 function,
 		F(ACE2) | F(ACE2_EN) | F(PHE) | F(PHE_EN) |
 		F(PMM) | F(PMM_EN);
 
-		F(ADX) | F(SMAP) | F(AVX512IFMA) | F(AVX512F) | F(AVX512PF) |
-		F(AVX512ER) | F(AVX512CD) | F(CLFLUSHOPT) | F(CLWB) | F(AVX512DQ) |
-		F(SHA_NI) | F(AVX512BW) | F(AVX512VL) | f_intel_pt;
-
 	/* cpuid 0xD.1.eax */
 	const u32 kvm_cpuid_D_1_eax_x86_features =
 		F(XSAVEOPT) | F(XSAVEC) | F(XGETBV1) | f_xsaves;
-- 
2.17.1

